roles/openshift_grafana/files/dashboards/node-exporter-full-dashboard.json

```diff
@@ -7209,7 +7209,7 @@
 				  "expr": "node_vmstat_nr_active_file{instance=~\"$node\"}",
 				  "format": "time_series",
 				  "intervalFactor": 2,
-				  "legendFormat": "Active_file - Active file memory memory pages",
```

Comment:
This would be removed in #10107, so in order to avoid conflicts lets not include this change

Reply:
Updated. Excluded grafana.